sdk/v2/data_test.go

```diff
@@ -1,4 +1,4 @@
-package sdk_test
+package v2
```

Comment:
The same as in `v1`, I would keep `v2_test`

Reply:
why? @MagdaPaj